python/ray/dashboard/modules/aggregator/tests/test_aggregator_agent.py

```diff
@@ -90,6 +90,32 @@ def get_addr():
     return EventAggregatorServiceStub(channel)
 
 
+@pytest.mark.parametrize("event_processing_enabled", [True, False])
+def test_receive_events(fake_timestamp, event_processing_enabled):
```

Comment:
this type of unit test is, to put it kindly, not very useful...

let's add an integration test for setting the flag e2e